io.github.JakubMelka.Pdf4qt.json

```diff
@@ -0,0 +1,49 @@
+{
+    "app-id": "io.github.JakubMelka.Pdf4qt",
+    "runtime": "org.kde.Sdk",
+    "runtime-version": "6.5",
+    "sdk": "org.kde.Sdk",
+    "command": "/app/bin/Pdf4QtViewerProfi",
+    "build-options": {
+	"build-args": [ "--share=network" ]
+	},
+    "finish-args": [
+        "--socket=x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--share=ipc",
+        "--filesystem=xdg-config/kdeglobals:ro"
+    ],
+    "cleanup": [
+        "/pdf4qt",
+        "/vcpkg"
+    ],
+    "modules": [
+        {
+            "name": "PDF4QT",
+            "buildsystem": "simple",
+            "build-commands": [
+                "./build_pdf4qt.sh"
```

Comment:
you can't do that. This script want to download things. This is NOT possible.

If you need any dependency they should be either sources for this module, or separate modules installed.

Reply:
So, is it not possible to use the standard VCPKG package system in some way? Would it be acceptable to prepare binary compiled dependent libraries into a separate GitHub repository, and then download and use them?

Reply:
if VCPKG list what it download and from where, then maybe, like an offline mode. There are solution for cargo (Rust), pip (Python), go, etc.  See https://github.com/flatpak/flatpak-builder-tools So someone need to contribute VCPKG support.

Or you can add the dependencies in the manifest. Like other packages do.

Reply:
It seems to be possible that VCPKG can run in offline mode, if packages are downloaded before. I will try it and update build system.

Reply:
I hope I have fixed it.